topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.md

```diff
@@ -0,0 +1,643 @@
+>    - {% icon param-file %} *"gdsc2 ccle targene pharmacology predictions"*: `gdsc2_ccle_targene_pharmacology_predictions.tsv` (output of **PAPAA: PanCancer targene cell line predictions** {% icon tool %})
+>    - {% icon param-file %} *"Filename list of compounds"*: `compounds_of_interest.txt` (Input dataset)
+{: .hands_on}
+
+> ### {% icon details %} PanCancer Aberrant Pathway Activity Analysis `targene_pharmacology.R` inputs
+>    - \--classifier_results: String of the location of classifier folder
+>    - \--compound: Filename of list of pharmacological compounds for evaluation
+{: .details}
+
+> ### {% icon comment %} Outputs
+>    - Scatter plots with visualizing drug response compared to GDSC targene classifier scores
+>    for GDSC1 pharmacological dataset (GDSC1_targene_all_drug_response.pdf)
+>    - Scatter plots with visualizing drug response compared to CCLE targene classifier scores
+>    for GDSC1 pharmacological dataset (GDSC1_ccle_targene_all_drug_response.pdf)
+>    - Scatter plots with visualizing drug response compared to GDSC targene classifier scores
+>    for GDSC2 pharmacological dataset (GDSC2_targene_all_drug_response.pdf)
+>    - Scatter plots with visualizing drug response compared to CCLE targene classifier scores
+>    for GDSC2 pharmacological dataset (GDSC2_ccle_targene_all_drug_response.pdf)
+{: .comment}
+
+> ### {% icon question %} Tutorial Questions
+>
+> 1. Can you build a classifier for tumor suppressor genes (TSG) combination in PI3K pathway?
+> 2. How did the PI3K_TSG model performed compared to PI3K_OG?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. Try building a model using (PTEN,PIK3R1,STK11 genes and BRCA,COAD,ESCA,HNSC,LGG,LUAD,LUSC,PRAD,READ,GBM,UCEC,UCS cancer-types)
+> > 2. Check the AUROC and AUPR values. 
+> >
+> {: .solution}
+>
+{: .question}
+
+# **Conclusions**
+{:.no_toc}
+
+In our tutorial, we were able to use machine learning based PI3K oncogene classifier in galaxy to distinguish tumor samples based upon PI3K activity. The classifier shows high accuracy in detecting aberrant PI3K activity in TCGA samples. The model output include top weighted genes that represent transcriptional signature of the tumors with selected gene mutations.  The pan model (all disease gene model) when compared to individual disease models have equal or higher performance for most of individual diseases. We also learned how to work with multi-omic datasets, and perform 5 fold cross-validation to output probability scores for each samples. Further, we can apply the trained models to make predictions on a external expression dataset provided and predict their mutational status. In addition we used our models to evaluated cell-lines pharmacological response against various compounds. This information will be helpful to improve personalized medicine approaches to various individuals. 
```

Comment:
Please quantify "high accuracy" by writing 80%

Reply:
@anuprulez thanks for the suggestion will add that